uobtheatre/bookings/models.py

```diff
@@ -554,6 +555,20 @@ def complete(self):
         self.save()
         self.send_confirmation_email()
 
+    @property
+    def web_tickets_path(self):
+        """Generates the path to the public tickets display page on the frontend for this booking"""
+        query_string = "?" + "&".join(
+            [
+                f"performanceID={to_global_id('PerformanceNode', self.performance.id)}",
+            ]
+            + [
+                f"ticketID={to_global_id('TicketNode', id)}"
+                for id in self.tickets.values_list("id", flat=True)
+            ]
+        )
+        return f"/user/booking/{self.reference}/tickets" + query_string
```

Comment:
We should use `urlencode(params, True)` here (from urllib.parse import urlencode)